wireguard/metrics.go

```diff
@@ -95,7 +95,7 @@ func (collector *Metrics) descsByDevice(d chan<- *prometheus.Desc, device *wgtyp
 
 func (collector *Metrics) descByPeer(d chan<- *prometheus.Desc, peer *wgtypes.Peer) {
 	if peer == nil {
-		collector.logCtx.Error("BUG: called descByPeer with nil peer")
+		collector.logCtx.Debug("BUG: called descByPeer with nil peer")
```

Comment:
same here - we should never hit this unless something is actually very wrong, so seems good to yell about it.